### ISSUE TITLE
[Bug Report] Rom not found after installing AutoRom

### ISSUE THREAD

**Issue by MrCley** [Farama-Foundation/PettingZoo#1160]
### Describe the bug

When trying to run the Atari examples in the docs, the same error for all games is shown:

> OSError: rom tennis is not installed. Please install roms using AutoROM tool (https://github.com/Farama-Foundation/AutoROM) or specify and double-check the path to your Atari rom using the `rom_path` argument.

It says that the rom is not installed and suggests installing AutoROM, which is already installed (pip list below).

Thanks! 

### Code example

```shell
from pettingzoo.atari import tennis_v3

env = tennis_v3.env(render_mode="human")
env.reset(seed=42)

for agent in env.agent_iter():
    observation, reward, termination, truncation, info = env.last()

    if termination or truncation:
        action = None
    else:
        # this is where you would insert your policy
        action = env.action_space(agent).sample()

    env.step(action)
env.close()
```

### System info

M1 macOs Ventura 13.0.1 
Python 3.11.6
pip list:

ale-py==0.8.1
AutoROM==0.6.1
AutoROM.accept-rom-license==0.6.1
certifi==2023.11.17
charset-normalizer==3.3.2
click==8.1.7
cloudpickle==3.0.0
Farama-Notifications==0.0.4
gymnasium==0.29.1
idna==3.6
importlib-resources==6.1.1
multi-agent-ale-py==0.1.11
numpy==1.26.3
pettingzoo==1.24.3
pygame==2.3.0
requests==2.31.0
Shimmy==0.2.1
tqdm==4.66.1
typing_extensions==4.9.0
urllib3==2.1.0

### Checklist

- [X] I have checked that there is no similar [issue](https://github.com/Farama-Foundation/PettingZoo/issues) in the repo

**Comment by elliottower** [comment]
Can you give the full stack trace? We have testing in our CI for both linux/mac and python 3.8-3.11 which successfully installs the ROMs and tests them, I’ll check what versions are installed there though just to be sure

**Comment by elliottower** [comment]
Oh what it might be is that we don’t explicitly test on M1 macs currently, but I believe that is now possible in GitHub actions so I could test it out

**Comment by MrCley** [comment]
@elliottower that was fast! 😄 
I've added the stack trace to the issue description. Thanks for looking into it!

**Comment by elliottower** [comment]
The tests all passed on the different Mac versions (11, 12, and 13 with m1) so I think it must be something on your end a

**Comment by pseudo-rnd-thoughts** [comment]
Could you rerun `pip install "gymnasium[accept-atari-roms]"` then run 
```python
import gymnasium
gymnasium.make("ALE/Pong-v5")
```

**Comment by MrCley** [comment]
> Could you rerun `pip install "gymnasium[accept-atari-roms]"` then run
> 
> ```python
> import gymnasium
> gymnasium.make("ALE/Pong-v5")
> ```

if I install the gymnasium rom license (_pip install 'gymnasium[accept-rom-license]'_), and try the _make()_ method, it does create the env.

But using the _tennis_v3.env()_ provided in the docs, it still gives the error provided in the description above.

**Comment by pseudo-rnd-thoughts** [comment]
@elliottower It seems like ale-py is doing something different from PettingZoo when finding the rom which is part of the issue

**Comment by jjshoots** [comment]
Adding on, a good place to start debugging this is probably [here](https://github.com/Farama-Foundation/PettingZoo/blob/master/pettingzoo/atari/base_atari_env.py#L93). I can start looking into this later.

**Comment by jjshoots** [comment]
@MrCley I've looked into this and for some reason I cannot replicate this. Is this still happening on your end?

**Comment by MrCley** [comment]
Hi @jjshoots, yes it still happens :( 

I've made another project (also Python 3.11), cloned the repo and ran the tests (`pip install -e .[testing,all]` then `pytest -v`), they pass except the Atari ones, which fail because of the missing ROM.

**Comment by jjshoots** [comment]
I assume you've done `AutoROM -v`?

**Comment by MrCley** [comment]
yes, I ran the tests both before and after running `AutoROM -v` and got the same results

**Comment by jjshoots** [comment]
@MrCley I assume you're using some form of virtual env? If so, could you print out the contents of `*/lib/python3.10/site-packages/multi_agent_ale_py/roms`?

**Comment by MrCley** [comment]
hey @jjshoots, just checked and there is no ROM folder in there:

![Screenshot 2024-02-03 at 15 05 03](https://github.com/Farama-Foundation/PettingZoo/assets/14186931/2acd3af3-8c26-4620-a8de-6fe43d285b28)

**Comment by jjshoots** [comment]
@MrCley Apologies for the repeated slow responses.
That does not look good...

Could you open up a Python interpreter within the venv, and run the following and send the output here?

```python
import importlib.resources as r
r.files("multi_agent_ale_py")
r.files("AutoROM")
```

**Comment by jkterry1** [comment]
Hey, I'm closing this due to inactivity and because I _think_ the issue no longer persists, if this still impacts a future traveler please create a new issue
